.changeset/custom-changelog.cjs

```diff
@@ -0,0 +1,64 @@
+/** @type {import('@changesets/types').ChangelogFunctions} */
```

Comment:
this does nothing
this jsdoc type import is used to annotate your fn, take a look at how presentation team does it